src/api/spec/features/webui/projects_spec.rb

```diff
@@ -15,20 +15,42 @@
     expect(page).to have_css('h3', text: project.title)
   end
 
-  it 'changing project title and description' do
-    login user
-    visit project_show_path(project: project)
+  describe 'changing project title and description' do
+    context 'when accepting the changes' do
+      it 'updates the project title, description and url' do
+        login user
+        visit project_show_path(project: project)
+
+        click_on('Edit Project')
```

Comment:
This works because the title of the link is still `Edit Project`. Since it's going to be changed, this needs to also change. 

```suggestion
        click_on('Edit')
```